storage/mysql/log_storage.go

```diff
@@ -367,9 +385,9 @@ func (t *logTreeTX) QueueLeaves(ctx context.Context, leaves []*trillian.LogLeaf,
 			return nil, fmt.Errorf("Unsequenced: %v", err)
 		}
 	}
+	queuedCounter.Add(float64(len(leaves)), labelForTX(t))
```

Comment:
I'll trust you know what you're doing with the changes here and below... :smiley: